README.md

```diff
@@ -1,6 +1,6 @@
 <div align="center">
   <h1>Fast React Static Renderer</h1>
-  <strong>A framework to build scalable and fast ecommerce platforms with thousands of products.</strong>
+  <strong>A framework to build scalable and fast ecommerce platforms with thousands of pages.</strong>
```

Comment:
Should we actually specify ecommerce here?

Reply:
I don't think so, it's not really ecommerce specific

Reply:
any suggestion?

Reply:
static content platforms? Idk maybe we should just leave it for now since FRSR is still evolving

Reply:
ok I'll leave it for now